src/ssh.c

```diff
@@ -1628,6 +1628,16 @@ int wolfSSH_CTX_SetBanner(WOLFSSH_CTX* ctx,
     return WS_SUCCESS;
 }
 
+WOLFSSH_API int wolfSSH_CTX_SetSshProtoIdStr(WOLFSSH_CTX* ctx,
```

Comment:
please remove the leading WOLFSSH_API macro here on function implementation and add a null check for 'ctx' before de-reference.

Reply:
done